DataFormats/src/Jet.cc

```diff
@@ -69,85 +72,106 @@ float Jet::scaleFactorCSVv2(Jet::BTAGCSV_CUT cutType, int syst, JETFLAV flav) co
+  if (cutType == 2 && syst == +1 && flav == 1 && pt > 140 && pt < 200 ) return (-(5.1345)+(0.000820101*(log(pt+11518.1)*(log(pt+11518.1)*(3-(-(8.66128*log(pt+11518.1))))))))+0.098963312804698944;
+  if (cutType == 2 && syst == +1 && flav == 1 && pt > 200 && pt < 300 ) return (-(5.1345)+(0.000820101*(log(pt+11518.1)*(log(pt+11518.1)*(3-(-(8.66128*log(pt+11518.1))))))))+0.14805065095424652;
+  if (cutType == 2 && syst == +1 && flav == 1 && pt > 300 && pt < 670 ) return (-(5.1345)+(0.000820101*(log(pt+11518.1)*(log(pt+11518.1)*(3-(-(8.66128*log(pt+11518.1))))))))+0.149765393555164337;
+  if (cutType == 2 && syst == +1 && flav == 0 && pt > 30 && pt < 50 ) return (-(5.1345)+(0.000820101*(log(pt+11518.1)*(log(pt+11518.1)*(3-(-(8.66128*log(pt+11518.1))))))))+0.031852148473262787;
+  if (cutType == 2 && syst == +1 && flav == 0 && pt > 50 && pt < 70 ) return (-(5.1345)+(0.000820101*(log(pt+11518.1)*(log(pt+11518.1)*(3-(-(8.66128*log(pt+11518.1))))))))+0.023946098983287811;
+  if (cutType == 2 && syst == +1 && flav == 0 && pt > 70 && pt < 100 ) return (-(5.1345)+(0.000820101*(log(pt+11518.1)*(log(pt+11518.1)*(3-(-(8.66128*log(pt+11518.1))))))))+0.038635428994894028;
+  if (cutType == 2 && syst == +1 && flav == 0 && pt > 100 && pt < 140 ) return (-(5.1345)+(0.000820101*(log(pt+11518.1)*(log(pt+11518.1)*(3-(-(8.66128*log(pt+11518.1))))))))+0.031439229846000671;
+  if (cutType == 2 && syst == +1 && flav == 0 && pt > 140 && pt < 200 ) return (-(5.1345)+(0.000820101*(log(pt+11518.1)*(log(pt+11518.1)*(3-(-(8.66128*log(pt+11518.1))))))))+0.049481656402349472;
+  if (cutType == 2 && syst == +1 && flav == 0 && pt > 200 && pt < 300 ) return (-(5.1345)+(0.000820101*(log(pt+11518.1)*(log(pt+11518.1)*(3-(-(8.66128*log(pt+11518.1))))))))+0.07402532547712326;
+  if (cutType == 2 && syst == +1 && flav == 0 && pt > 300 && pt < 670 ) return (-(5.1345)+(0.000820101*(log(pt+11518.1)*(log(pt+11518.1)*(3-(-(8.66128*log(pt+11518.1))))))))+0.074882696777582169;
+  if (cutType == 0 && syst == 0 && flav == 2 && pt > 20 && pt < 1000 ) return ((1.07278+(0.000535714*pt))+(-1.14886e-06*(pt*pt)))+(7.0636e-10*(pt*(pt*pt)));
+  if (cutType == 0 && syst == -1 && flav == 2 && pt > 20 && pt < 1000 ) return ((1.01637+(0.000265653*pt))+(-4.22531e-07*(pt*pt)))+(2.23396e-10*(pt*(pt*pt)));
+  if (cutType == 0 && syst == +1 && flav == 2 && pt > 20 && pt < 1000 ) return ((1.12921+(0.000804962*pt))+(-1.87332e-06*(pt*pt)))+(1.18864e-09*(pt*(pt*pt)));
+  if (cutType == 1 && syst == 0 && flav == 2 && pt > 20 && pt < 1000 ) return 1.14022;
+  if (cutType == 1 && syst == -1 && flav == 2 && pt > 20 && pt < 1000 ) return 0.94022;
+  if (cutType == 1 && syst == +1 && flav == 2 && pt > 20 && pt < 1000 ) return 1.34022;
+  if (cutType == 2 && syst == 0 && flav == 2 && pt > 20 && pt < 1000 ) return 0.907317;
+  if (cutType == 2 && syst == -1 && flav == 2 && pt > 20 && pt < 1000 ) return 0.557317;
+  if (cutType == 2 && syst == +1 && flav == 2 && pt > 20 && pt < 1000 ) return 1.257317;
```

Comment:
Should be pt >= 20 etc.
Do you think that this form is better than the previous one?